website/identifiers/clients/base.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+import requests
+
+from framework.exceptions import HTTPError
+from website import settings
+
+
+class AbstractIndentifierClient(object):
+
+    def __init__(self, base_url, prefix):
+        self.base_url = base_url
+        self.prefix = prefix
+
+    def build_metadata(self, object):
+        """ Build the metadata object used to register the object
+        with the specified client"""
+        raise NotImplementedError()
+
+    def build_doi(self, object):
+        """Method this client uses to build a DOI"""
+        return settings.DOI_FORMAT.format(prefix=self.prefix, guid=object._id)
+
+    def _make_request(self, method, url, **kwargs):
+        expects = kwargs.pop('expects', None)
+        throws = kwargs.pop('throws', None)
```

Comment:
Are these used, or this copy-pasta from `BaseClient`? Can these be removed?

Reply:
Looking further, I'm not sure `_make_request` is necessary at all. `DataciteClient` doesn't use it at all, and if you remove the above two, lines, then the method is equivalent to using `requests.request`.